e2e/node_config.go

```diff
@@ -157,7 +157,7 @@ func baseTemplateLinux(location string) *datamodel.NodeBootstrappingConfiguratio
 						UserAssignedClientID:              "",
 						CustomHyperkubeImage:              "",
 						CustomKubeProxyImage:              "mcr.microsoft.com/oss/kubernetes/kube-proxy:v1.26.0.1",
```

Comment:
@r2k1 could we potentially resolve these versions dynamically based on the e2e cluster version?